src/NuGet.CommandLine/Commands/RestoreCommand.cs

```diff
@@ -324,7 +323,7 @@ private void ReadSettings(PackageRestoreInputs packageRestoreInputs)
                 {
                     var message = string.Format(
                         CultureInfo.CurrentCulture,
-                        "RestoreCommandFileNotFound",
+                        LocalizedResourceManager.GetString("RestoreCommandFileNotFound"),
```

Comment:
That's a good idea, but both of these items should be done across the project to keep things consistent. I see over 100 instances of it being done the way it currently is, and 2 using name of and CommandLineException. Changing it up randomly will be confusing here.